bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/navigation/BootstrapPagingNavigator.java

```diff
@@ -60,6 +63,39 @@ public BootstrapPagingNavigator(final String markupId, final IPageable pageable,
     }
 
     @Override
+    protected void onInitialize() {
+        super.onInitialize();
+        add(new TransparentWebMarkupContainer("firstItem")
+                .add(new CssClassNameAppender(new AbstractReadOnlyModel<String>() {
+                    @Override
+                    public String getObject() {
+                        return get("first").isEnabled() ? null : "disabled";
+                    }
+                })));
+        add(new TransparentWebMarkupContainer("prevItem")
+                .add(new CssClassNameAppender(new AbstractReadOnlyModel<String>() {
+                    @Override
+                    public String getObject() {
+                        return get("prev").isEnabled() ? null : "disabled";
+                    }
+                })));
+        add(new TransparentWebMarkupContainer("nextItem")
+                .add(new CssClassNameAppender(new AbstractReadOnlyModel<String>() {
+                    @Override
+                    public String getObject() {
+                        return get("next").isEnabled() ? null : "disabled";
+                    }
+                })));
+        add(new TransparentWebMarkupContainer("lastItem")
+                .add(new CssClassNameAppender(new AbstractReadOnlyModel<String>() {
+                    @Override
+                    public String getObject() {
+                        return get("last").isEnabled() ? null : "disabled";
+                    }
+                })));
```

Comment:
Create a new component (private static class) for this need instead of copy/pasting the same code 4 times.